.changeset/wise-flies-laugh.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-home': minor
```

Comment:
I thiiink as `filterBy` is optional, this can be a patch. I'll just double check with @backstage/maintainers